### PR TITLE
Flux: update maintainers list and source file

### PR DESCRIPTION
**Pull request by scottrigby**
We now have a combined maintainers list for the Flux project (added in https://github.com/fluxcd/community/pull/134). 
This PR updates the CNCF list.

### CODE REVIEW

**Review by idvoretskyi — APPROVED** [review]

**Comment by idvoretskyi** [comment]
PTAL @amye

File: project-maintainers.csv [hunk 1]
```diff
@@ -383,9 +383,26 @@ Incubating,Thanos,Bartłomiej Płotka,Red Hat,bwplotka,https://github.com/thanos
 ,,Giedrius Statkevičius,,GiedriusS,
 ,,Povilas Versockas,,povilasv,
 ,,Dominic Green,Improbable,domgreen,
-Incubating,Flux,Hidde Beydals,Weaveworks,hiddeco,https://github.com/fluxcd/flux/blob/master/MAINTAINERS
+Incubating,Flux,Hidde Beydals,Weaveworks,hiddeco,https://github.com/fluxcd/community/blob/main/flux-project-maintainers.yaml
 ,,Michael Bridgen,Weaveworks,squaremo,
 ,,Stefan Prodan,Weaveworks,stefanprodan,
+,,Aurel Canciu, Sortlist,relu,
```
Comment (dholbach) [inline project-maintainers.csv:389]:
```suggestion
,,Aurel Canciu, NexHealth,relu,
```

@relu - right?

File: project-maintainers.csv [hunk 2]
```diff
@@ -383,9 +383,26 @@ Incubating,Thanos,Bartłomiej Płotka,Red Hat,bwplotka,https://github.com/thanos
...
+,,Max Jonas Werner, D2iQ,makkes,
+,,Philip Laine, Xenit,phillebaba,
+,,Sunny, Weaveworks,darkowlzz,
+,,Alison Dowdney,,alisondy,
+,,Daniel Holbach, Weaveworks,dholbach,
+,,Scott Rigby, Weaveworks,scottrigby,
+,,Takeshi Yoneda, DMM.com,mathetake,
+,,Sean Eagan, AT&T,seaneagan,
```
Comment (dholbach) [inline project-maintainers.csv:397]:
```suggestion
,,Sean Eagan, Microsoft,seaneagan,
```

@seaneagan - right?

File: project-maintainers.csv [hunk 3]
```diff
@@ -383,9 +383,26 @@ Incubating,Thanos,Bartłomiej Płotka,Red Hat,bwplotka,https://github.com/thanos
...
+,,Somtochi Onyekwere, Weaveworks,SomtochiAma,
+,,Dinos Kousidis, Independent,dinosk,
```
Comment (dholbach) [inline project-maintainers.csv:399]:
```suggestion
,,Dinos Kousidis, CrowdStrike,dinosk,
```

@dinosk - right?

File: project-maintainers.csv [hunk 4]
```diff
@@ -383,9 +383,26 @@ Incubating,Thanos,Bartłomiej Płotka,Red Hat,bwplotka,https://github.com/thanos
...
 ,,Michael Bridgen,Weaveworks,squaremo,
 ,,Stefan Prodan,Weaveworks,stefanprodan,
+,,Aurel Canciu, Sortlist,relu,
+,,Max Jonas Werner, D2iQ,makkes,
+,,Philip Laine, Xenit,phillebaba,
+,,Sunny, Weaveworks,darkowlzz,
+,,Alison Dowdney,,alisondy,
+,,Daniel Holbach, Weaveworks,dholbach,
+,,Scott Rigby, Weaveworks,scottrigby,
+,,Takeshi Yoneda, DMM.com,mathetake,
```
Comment (dholbach) [inline project-maintainers.csv:396]:
```suggestion
,,Takeshi Yoneda, Tetrate,mathetake,
```

@mathetake - right?

File: project-maintainers.csv (hunk 2 above)
Comment (dholbach) [reply project-maintainers.csv:397]: https://github.com/fluxcd/helm-controller/blob/main/MAINTAINERS might need updating as well.

File: project-maintainers.csv (hunk 3 above)
Comment (dholbach) [reply project-maintainers.csv:399]: https://github.com/fluxcd/go-git-providers/blob/main/MAINTAINERS might need updating as well.

File: project-maintainers.csv (hunk 4 above)
Comment (dholbach) [reply project-maintainers.csv:396]: https://github.com/fluxcd/flagger/blob/main/MAINTAINERS might need updating as well.

File: project-maintainers.csv (hunk 1 above)
Comment (dholbach) [reply project-maintainers.csv:389]: https://github.com/fluxcd/flux2/blob/main/MAINTAINERS might need updating as well.

**Comment by amye** [comment]
Will wait to merge until suggestions are resolved.

File: project-maintainers.csv (hunk 3 above)
Comment (dholbach) [reply project-maintainers.csv:399]: https://github.com/dinosk confirms.

File: project-maintainers.csv (hunk 4 above)
Comment (dholbach) [reply project-maintainers.csv:396]: https://github.com/mathetake confirms.

File: project-maintainers.csv (hunk 1 above)
Comment (dholbach) [reply project-maintainers.csv:389]: https://github.com/relu confirms.

Comment (relu) [reply project-maintainers.csv:389]: Yes, this is correct.

Comment (scottrigby) [reply project-maintainers.csv:389]: Confirmed from Linkedin: https://www.linkedin.com/in/aurelcanciu/. Sortlist was a previous role.

File: project-maintainers.csv (hunk 2 above)
Comment (scottrigby) [reply project-maintainers.csv:397]: Yep will do

File: project-maintainers.csv (hunk 3 above)
Comment (scottrigby) [reply project-maintainers.csv:399]: Linkedin also matches 👍 https://www.linkedin.com/in/kousidis/

Will also update this corresponding MAINTAINERS file

**Comment by scottrigby** [comment]
Thanks @dholbach and @amye - resolved!

File: project-maintainers.csv (hunk 3 above)
Comment (dinosk) [reply project-maintainers.csv]: Yes, that's correct 👍

File: project-maintainers.csv (hunk 4 above)
Comment (mathetake) [reply project-maintainers.csv]: 👍

**Pull request merged**